scripts/release/workflow/publish.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+package_name() {
+  echo $(node --print --eval "require('./package.json').name")
+}
+
+latest_npm_version() { 
+  echo $(npm info "$package_name" version)
+}
+
+package_json_version() {
+  echo $(node --print --eval "require('./package.json').name")
+}
+
+dist_tag() {
+  if [ $PRERELEASE = "true" ]; then
+    echo "next"
+  else
+    if [ `npx semver -r ">$package_json_version" $latest_npm_version` = "" ]; then
+      echo "latest"
+    else
+      # This is a patch for an older version
+      # npm can't publish without a tag
+      echo "tmp"
+    fi
+  fi
+}
+
+publish() {
+  cd contracts
```

Comment:
We don't really need this anymore now that we use the tarball.

Reply:
Not sure about this, if it's at the root, doesn't it publish to `openzeppelin-contracts` instead? I'm going with the most certain config to avoid testing runs.

If you can confirm it's fine I'll move it

Reply:
The tarball includes a `package.json` inside of it that corresponds to a specific package, in our case `@openzeppelin/contracts`. Tthis is what gets published.

Reply:
Solved in [this commit](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/3915/commits/3f91831e9007b6df023d70608417d203f9b12b80)